Documentation/Signoffs/detaildesign_camera.md

```diff
@@ -0,0 +1,121 @@
+    - The worst case scenario is that of in which a Tacoma is entering Sector 6 near the tree and a Sentra is exiting Sector 6 near the tree, in which for a time period the tall vehicle will obstruct the short vehicle. (We are assuming typical American road structures, in which you drive on the right side of the road, so entering the lot makes the car will occur closer to the camera and exiting the lot makes the car further from the camera.) However, due to the relative frames, because the vehicle entering the lot is very close to the camera, it will be on the video feed for a much shorter than the the vehicle exiting the lot, so it is highly probable that since a 31° FOV will lead to at least 56 ft of the visible parking spaces visible (and thus the demarcation in which a car can move through that entrance), the worst-case scenario will work itself out as the exiting short car will still be on the feed once the entering tall car is accounted for.
+  * Attachment to Light Posts
+    - Because drilling into posts is not allowed (as per Facilities), hose clamps will need to be used to mount the cameras. Please refer to Mounting for more information.
+  * Casing
+    - The casing must protect the cameras from vandalism, weather, birds, etc.
+    - The casing also cannot obstruct the view of the lens of the cameras.
+    - The casing will be designed by the Mechanical Engineering team. (If not, then the ECE will create the first prototypes of the casing to be later improved by the ME team.)
+  * Day vs. Night Performance
+    - The cameras are expected to perform properly during the day. However, because it has been specified that the entire system should only output the number of available spots during parking enforcement hours, which fall in the day with the exception of a few minutes during the Winter months (due to Daylight Savings Time bringing an early sundown), the cameras do not have to have night vision.
+
+2.  Monitoring of Parking Spaces and Entrances
+  * Sectoring of Bell Hall Red Lot
+    - The parking lot behind Bell Hall has 78 Red parking spots. However, many surveillance systems and other vehicle detection systems have cameras that track a small number of spots, so it is reasonable to sector Bell Hall and have cameras fixated on certain sectors/spots.
+    - Because of the lack of mountable spots for cameras surrounding the Bell Hall lot, it has been decided that multiple cameras will need to be placed to view certain sectors of the lot. Thus, Bell Hall has been apportioned into six sectors in a manner to balance the performance of the cameras and the costs since cameras are not inexpensive. The sectors were selected to ensure that if a sector is a statically tracked area (see below) that all parking spots are visible and that if a sector is a dynamically tracked area that entrances can be monitored properly.
+  * Statically vs. Dynamically Tracked Parking Sector
```

Comment:
I still don't see any constraints that deal with the dynamic application. They are not the same. You have the pixel density etc for the static system and that seems in order. But what are the considerations for the dynamic application?

Also, it doesn't seem you have considered if you can get data out of the camera. (I think you have considered, but it isn't discussed).